provisioning/wildfly_channel_in_zips.adoc

```diff
@@ -0,0 +1,210 @@
+We guarantee that minor and micro releases of WildFly do not contain breaking changes and this extends to their channels.
+
+=== Tooling Workflow
+
+With that in mind, we propose to incorporate the channel metadata in the WildFly distributions that describes the exact version of the channel used to provision the server.
+In practice, that means that the installation would be "bound" to a versioned channel (eg `org.wildfly.channels:wildfly:33.0.0.Final`).
+
+Updating to a more recent version is a user decision and would require the user to give as input the version of the WildFly they want to _update to_ (e.g. `33.0.1.Final` or `34.0.0.Final`).
+After the update is successful, the channel metadata would be updated to point to the current version of the installation.
+
+WildFly must contain the tooling to let user update installations without having to download another piece of software.
+
+The tooling should be made available in the `bin` directory of this installation and affects only the parent directory (`JBOSS_HOME` would have no bearing for update).
+
+The tooling would need to provide the following operations:
+
+* List the actual provisioned version of the installation (the channel it was provisioned from, read from the metadata added by WFLY-19221)
+** Informational only - no changes to the installation state
+* List the available updates for the installations (can be obtained by querying Maven Central to get all the versions for the channel artifact based on the maven-metadata.xml)
+** Informational only - no changes to the installation state
+* Apply an update to a more recent version of WildFly (based on the list mentioned above)
+** Verify that the installation is not running.
+** The changes that would be applied to the installation are shown to the user
+** After their confirmation, the update is applied and the installation state changes
+** After the update, the provisioned version of the installation is the more recent version that was applied
+
+Since https://issues.redhat.com/browse/WFCORE-6206[WFCORE-6206], there is `installation-manager` script that is meant for internal usage. This script could be repurposed to provide these operations to the user.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-19221[WFLY-19221] - [Preview] Incorporate channel metadata in the download zips
+* https://github.com/wildfly-extras/prospero/issues/754[prospero#754] - Add ability to specify a "target" version for updates
+* https://issues.redhat.com/browse/WFMP-271[WFMP-271] Add a <name> parameter to the <channel> configuration
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19130[WFLY-19130] - [Community] Publish Wildfly channel and manifest during project build
+** http://docs.wildfly.org/wildfly-proposals/build/WFLY-19130_publish_Wildfly_channel_manifest.html[Analysis Document]
+
+=== Stability Level
+
+* [x] Experimental
```

Comment:
I have only had a very fast scan so sorry if I jumped over it, if this is an experimental feature - how does a user turn on this experimental feature / acknowledge that they are using an experimental feature rather than assuming it is a community feature?